doc_source/reference-pipeline-structure.md

```diff
@@ -150,13 +150,9 @@ The numbering of serial actions do not have to be in strict sequence\. For examp
 
   For a full list of valid action types, see the [AWS CodePipeline API Reference](http://docs.aws.amazon.com/codepipeline/latest/APIReference)\.
 
-+ Valid provider types for an action category depend on the category\. For example, for a source action type, a valid provider type is: 
++ Valid provider types for an action category depend on the category\. For example, for a source action type, a valid provider type is 'S3', 'GitHub', or 'CodeCommit'. 
```

Comment:
Thank you very much for your contribution. These comments will make the documentation much more helpful. I am in the process of merging these comments now. I should be able to re-post by tomorrow.